lit-base/model_service.py

```diff
@@ -0,0 +1,92 @@
+import mxnet as mx
+from model_loader import sym, arg_params, aux_params
+
+import tempfile
+import urllib2
+
+import numpy as np
+from PIL import Image
+from StringIO import StringIO
+
+
+ctx = mx.cpu()
+
+
+def preprocess(event):
+    """
+    Preprocess Lambda event into input format for mx mod.
+    :return: input NDArray
+    event : dict
+        Lambda event
+    """
+    img = None
+    try:
+        # GET method
+        if event.has_key('httpMethod') and event['httpMethod'] == 'GET':
```

Comment:
use event.get('httpMethod') instead - there's no need to check for has_key

Reply:
please use get() method instead of retrieving key directly from dict. This prevents the need for checking for KeyError & you can pass in a default value incase if there's no value present for a given key - 
For example something like this - `event.get('httpMethod', '')`